### ISSUE TITLE
Support for SublimeText v4?

### ISSUE THREAD

**Issue by gtdeng** [xsleonard/sublime-MarkdownImages#15]
Yes I do realize that the package description is for Sublime Text v3, but it would be extremely helpful to migrate this to work on v4. 🙏

**Comment by xsleonard** [comment]
Did v4 break all of the existing packages?

**Comment by gtdeng** [comment]
Yup; post migration Sublime ignored the incompatible plugins such as MarkdownImages.

**Comment by R4wd0g** [comment]
i'm facing same problem too. Sometimes when invoke the plugin, sublime stucks. Just to confirm: is this plugin compatible with v4 nowadays? thanks

**Comment by Unknow0059** [comment]
Would be nice if it worked on ST4, I love this plugin.

Oops. I can confirm it does work.
I had another package conflicting with it, it was ImagePaste. Though, 'Image Link Insert' might also conflict with it.
The two just insert the file path, not render any images, but they use the same syntax.

Pasting the image as a url is indispensable for this plugin.
